scripts/build-docs.sh

```diff
@@ -21,12 +21,10 @@ appledoc \
     --no-warn-invalid-crossref \
     --no-warn-undocumented-object \
     --no-warn-undocumented-member \
-    --ignore "Realm/RLMRealm_Dynamic.h" \
-    --ignore "Realm/RLMArrayAccessor.h" \
-    --ignore "Realm/RLMArrayAccessor.mm" \
-    --ignore "Realm/RLMQueryUtil.h" \
-    --ignore "Realm/RLMUtil.h" \
-    --ignore "Realm/RLMRealm_Dynamic.h" \
+    --ignore ".m" \
+    --ignore ".mm" \
+    --ignore "Realm/RLMObjectStore.h" \
+    --ignore "Realm/RLMCollection.h" \
```

Comment:
I wasn't sure on that. The current API docs have it public but only setValue and valueForKey is actually documented by appledoc. I added the proper documentation for appledoc to generate, but didn't know if we wanted to actually include it. Given that it was incomplete, I assumed it might have been an oversight to have it public.